### ISSUE TITLE
[Eclipse-ThirdParty][A11yMAS][Accessibility][Keyboard][MAS2.4.3][Eclipse][Wipro]:Keyboard focus order is incorrect when ACR Explorer window opens in editor.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1417]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. In "Azure Explorer" , select the pushed image under container registries and press shift+F10 to open the context menu.
4. Navigate to and select "Open ACR Explorer" menu item and hit enter then observe the keyboard focus

**Expected Result:**
When " Open ACR Explorer" is opened after hitting 'Tab' keyboard focus should be on the first interactive element in the window.

**Actual Result:**
Keyboard focus is in New button which is present in top of the page

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version:** Version 1709(Os Build 16299.192)

![keyboard focus order is illogical when acr explorer window opens](https://user-images.githubusercontent.com/22956130/35717342-a4a71bca-0804-11e8-9de8-ee4ab790905b.jpg)

**Comment by Eskibear** [comment]
It is determined by Eclipse IDE. We cannot fix it.

**Comment by jdneo** [comment]
Fixed

**Comment by Eskibear** [comment]
fixed in #1605 , will be in next release

**Comment by Eskibear** [comment]
already released in v3.6.0

**Comment by WiproPSAccTest** [comment]
Verified the issue in V 3.6.0.
Issue got fixed

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749871
